test/unit/references/__init__.py

```diff
@@ -0,0 +1,41 @@
+import mongomock
+
+
+mock_mongo_client = mongomock.MongoClient()
+
+database = mock_mongo_client.get_database("fhirstore")
+database.create_collection("Patient")
+database.create_collection("HealthcareService")
+
+database["Patient"].insert_many(
+    [
+        {
+            "_id": "987654321",
+            "id": "123456",
```

Comment:
maybe you could define this variable externally to avoid hard-coding it everywhere